eng/common/scripts/update-docs-metadata.ps1

```diff
@@ -70,12 +70,15 @@ function GetAdjustedReadmeContent($pkgInfo, $lang){
     }
 
     $fileContent = $pkgInfo.ReadmeContent
-    $foundTitle = ""
 
     # only replace the version if the formatted header can be found
-    $titleRegex = "(\# Azure .+? (?:client|plugin|shared) library for (?:JavaScript|Java|Python|\.NET|C))"
-    $fileContent = $pkgInfo.ReadmeContent -replace $titleRegex, "`${1} - Version $($pkgInfo.PackageVersion) `n"
-    
+    $titleRegex = "(\#.*(?<filetitle>Azure .+? (?:client|plugin|shared) library for (?:JavaScript|Java|Python|\.NET|C)))"
```

Comment:
```suggestion
    $titleRegex = "(\#\s+(?<filetitle>Azure .+? (?:client|plugin|shared) library for (?:JavaScript|Java|Python|\.NET|C)))"
```
I know I'm the one that suggested `.*` but I think `\s+` for one more spaces is more appropriate then any chars.

Reply:
Exactly, thanks for catching this.